src/structures/APIMessage.js

```diff
@@ -176,6 +176,23 @@ class APIMessage {
       flags = this.options.flags != null ? new MessageFlags(this.options.flags).bitfield : this.target.flags.bitfield;
     }
 
+    let mentionUserIds;
+    let mentionRoleIds;
+    if (this.options.allowedMentions) {
+      if (this.options.allowedMentions.users) {
+        mentionUserIds = this.options.allowedMentions.users.map(u => this.target.client.users.resolveID(u));
```

Comment:
`target.client` can also be a [`WebhookClient`](https://discord.js.org/#/docs/main/stable/class/WebhookClient) which does not have a `users` property.

I'm not sure how to go about this.

Reply:
Dropped support for Resolvables